js/app/controllers/content-product.ng

```diff
@@ -138,7 +138,158 @@ mkdir my-new-bucket</pre>
+      the <cmdlet-ref name="New-BqSchema"></cmdlet-ref> and <cmdlet-ref name="Set-BqSchema"></cmdlet-ref>
+      cmdlets. New-BqSchema can take the formats for rows as parameters directly or as a JSON array of 
+      row descriptions. The results of New-BqSchema are always passed into Set-BqSchema which can either output
+      a Schema object or assign the Schema to an existing Table.  
+    </p>
+    <pre>
+# Assigns a Schema to $table
+$table = Get-BqTable "my_table" -DatasetId "my_dataset"
+New-BqSchema "Author" "STRING" | New-BqSchema "Copyright" "STRING" | 
+    New-BqSchema "Title" "STRING" | Set-BqSchema $table
+
+# Creates a schema object to be used in multiple tables.
+$schema = New-BqSchema "Author" "STRING" | New-BqSchema "Copyright" "STRING" | 
+          New-BqSchema "Title" "STRING" | Set-BqSchema $table
+    </pre>
+    <p>
+      Schema objects can be passed as parameters in Table creation if they are created ahead of time.
+    </p>
+    <pre>
+# Creates a new table with the Schema object from above.
+$table = $dataset | New-BqTable "my_table" -Schema $schema
+    </pre>
+    
+    <h2>TableRow</h2>
+    <p>
+      Data is added and removed from Tables in Rows. These Rows are accessible using the 
+      <cmdlet-ref name="Add-BqTableRow"></cmdlet-ref> and <cmdlet-ref name="Get-BqTableRow"></cmdlet-ref>
+      cmdlets. Add-BqTableRow takes CSV, JSON, and AVRO files to import into BigQuery.
+    </p>
+    <pre>
+# Ingests a CSV file and appends its rows onto the table 'my_dataset:my_table'.
+$table = New-BqTable "my_table" -DatasetId "my_dataset"
+$table | Add-BqTableRow CSV $filename -SkipLeadingRows 1 -WriteMode WriteAppend
+
+# Returns a list of the rows in 'book_data:classics'.
+$list = Get-BqTable "classics" -DatasetID "book_data" | Get-BqTableRow
+    </pre>
+    
+    <h2>Jobs</h2>
+    <p>
+      There are four types of jobs: Query, Load, Extract, and Copy. Query jobs run SQL style queries and 
+      output results to tables. Load jobs import Google Cloud Storage files into BigQuery. Extract jobs 
+      export BigQuery tables to GCS. Copy jobs copy an existing table to another new or existing table.
+      <cmdlet-ref name="Start-BqJob"></cmdlet-ref> starts any of these kinds of jobs as an asynchronous 
+      operation. Use the <code>-PollUntilComplete</code> flag to have the cmdlet block until the job is done.
+      <cmdlet-ref name="Receive-BqJob"></cmdlet-ref> will return the results of a query job once it is 
+      finished. <cmdlet-ref name="Get-BqJob"></cmdlet-ref> will return a reference object detailing the
+      current state and statistics on the job.  <cmdlet-ref name="Stop-BqJob"></cmdlet-ref> will send a 
+      request to the server to stop a certain job, and then returns immediately.
+    </p>
+    <pre>
+# Query Job: starts a query and outputs results into $table.
+Start-BqJob -Query "SELECT * FROM ``my_dataset:my_table``" -Destination $table
```

Comment:
Is it really "``my_dataset:table``"? That seems strange. Is there another way we can escape/unescape the string in PowerShell? Or is that just how BigQuery rolls?